types/stripe/index.d.ts

```diff
@@ -4572,6 +4597,9 @@ declare namespace Stripe {
             use_stripe_sdk: any;
         }
 
+        /** Payment methods supported by Payment Intents. This is a subsetset of all Payment Method types. See https://stripe.com/docs/api/payment_methods/create#create_payment_method-type */
+        type PaymentIntentPaymentMethodType = 'card' | 'card_present';
```

Comment:
nit: use double quotes.

Reply:
I was trying to follow the style within the `paymentIntent` namespace, which appears to be single quotes, but I didn't even manage that unfortunately (like at line 4508). The file uses a mix of single and double quotes. Do you think it'd be a good idea to add a lint rule to standardize it?

Reply:
Ah, gotcha - not blocking. I'm not too familiar with how linting works in this repo so don't feel too strongly, but in any case that would be a task for a separate PR.